db/models/news-models

```diff
@@ -0,0 +1,22 @@
+const db = require("../connection");
+
+const selectTopics = (slug = null) => {
+  const queryValues = [];
+
+  let baseSqlString = `SELECT * FROM topics `;
+
+  if (slug) {
+    baseSqlString += `WHERE slug = $1;`;
+    queryValues.push(slug);
+  }
+
+  return db.query(baseSqlString, queryValues).then((result) => {
+    if (result.rows.length === 0) {
+      return Promise.reject({ status: 404, msg: "Resource not found" });
+    } else {
+      return result.rows;
+    }
+  });
```

Comment:
As mentioned above. You will want to be considering things like queries later on but for this ticket we are only looking at getting an array of all the topics.